README.md

```diff
@@ -1,5 +1,34 @@
 # spark-commons
 
+
+[![Build](https://github.com/AbsaOSS/spark-hats/workflows/Build/badge.svg)](https://github.com/AbsaOSS/spark-commons/actions)
+
+sbt
+```scala
+libraryDependencies += "za.co.absa" %% "spark-commons" % "0.1.1"
```

Comment:
Wouldn't be better instead of hardcoded value, which will become obsolete with next released update. What about putting just a placeholder there?
Or can some plugin take of it? 🤔

Reply:
We can do like https://github.com/scopt/scopt

Do X.Y.Z and have a maven central badge.

And single badge at the top should be enough